PSPDFKit/Cordova-iOS
# Rename plugin from "com.pspdfkit.cordovaplugin" to "pspdfkit-cordova-ios"

## How to Test:

* Make sure you replace all occurrences of `https://github.com/PSPDFKit/Cordova-iOS.git` with `https://github.com/PSPDFKit/Cordova-iOS.git#rad/rename` in the local sample projects so that they use this branch instead of master.

* For testing in newly created apps, use `cordova plugin add https://github.com/PSPDFKit/Cordova-iOS.git#rad/rename` or `ionic cordova plugin add https://github.com/PSPDFKit/Cordova-iOS.git#rad/rename`
 
## What to Test:

- [x] Test that both bundled sample projects (`CordovaDemo` and `IonicDemo`) work greak.
- [x] Test with a newly created Cordova app
- [x] Test with a newly created Ionic app
  
  ## TODO:

- [ ] Publish the newly renamed package to npm
- [ ] Deprecate https://www.npmjs.com/package/com.pspdfkit.cordovaplugin
